From 4844eee9861a83078fed05da529f2242b2b47a92 Mon Sep 17 00:00:00 2001
From: Sergey Fedorov <vital.had@gmail.com>
Date: Fri, 28 Aug 2026 11:27:59 +0800
Subject: [PATCH 1/2] yt_http.h: fix headers order

Some macOS versions need sys/socket.h prior to netinet stuff. Inconsequential for other systems.
---
 include/ytcui-dl/yt_http.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git include/ytcui-dl/yt_http.h include/ytcui-dl/yt_http.h
index b87eccb..3fc485b 100644
--- include/ytcui-dl/yt_http.h
+++ include/ytcui-dl/yt_http.h
@@ -33,11 +33,11 @@
 
 #include <fcntl.h>
 #include <netdb.h>
+#include <sys/socket.h>
+#include <sys/types.h>
 #include <netinet/in.h>
 #include <netinet/tcp.h>
 #include <poll.h>
-#include <sys/socket.h>
-#include <sys/types.h>
 #include <unistd.h>
 
 #include <openssl/err.h>
-- 
2.55.0

